backend/src/main/java/ai/verta/modeldb/utils/ModelDBHibernateUtil.java

```diff
@@ -561,109 +562,37 @@ public static Query getWorkspaceEntityQuery(
    */
   @SuppressWarnings("unchecked")
   public static void runMigration(DatabaseConfig config)
-      throws ClassNotFoundException, ModelDBException {
+      throws ClassNotFoundException, ModelDBException, DatabaseException, SQLException {
     RdbConfig rdb = config.RdbConfiguration;
 
-    App app = App.getInstance();
-    Map<String, Map<String, Object>> migrationTypeMap =
-        (Map<String, Map<String, Object>>) app.getPropertiesMap().get(ModelDBConstants.MIGRATION);
-    if (migrationTypeMap != null && migrationTypeMap.size() > 0) {
-      new Thread(
```

Comment:
this is added because we are running migration on the background to service liveness for the kubernetes so added second thread for that which is not added by you. are you expecting all the migration are run on main thread and do not service up without finish?

Reply:
Yes. The migration is now run with the liquibase changes. We shouldn't run the migration in the background to avoid database inconsistencies

Reply:
Ok, then it would be fine.